_saas-integrations/marketo/marketo-latest.md

```diff
@@ -49,7 +49,7 @@ loading-reports: true
 # -------------------------- #
 
 feature-summary: |
-  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+  Stitch's {{ integration.display_name }} integration extracts data using the Bulk API, and replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
```

Comment:
Let's add the Bulk API part to the `api` property on this page. So, the `api` property should read like this: `[REST API](link-to-rest-api-docs) and [Bulk API](link-to-bulk-api-docs)`

And then there won't be any need to change this part!